internal/controller/server/adapter/dispatcher.go

```diff
@@ -20,18 +20,17 @@ type dispatcherResponse struct {
 	GameEnded int    `json:"end,omitempty"`
 }
 
-type DispatcherMsgConverter struct {
-	dispatcherMsg *dispatcherMsg
+func Unpack(message []byte) (*dispatcherMsg, error) {
```

Comment:
I see a little schizophrenia in this area :) 

https://github.com/HackYourCareer/SmartKickers/pull/63/files

Reply:
😅